uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt5/client/paho/MqttConnectionImpl.java

```diff
@@ -381,6 +395,9 @@ public void messageArrived(String topic, MqttMessage mqttMessage) {
             if (payloadFormatIndicator != null) {
                 logger.atInfo().log("Received MQTT message has payload format indicator '{}'", payloadFormatIndicator);
             }
+            if (messageExpiryInterval != null) {
+                logger.atInfo().log("Received MQTT message has message expiry interval '{}'", messageExpiryInterval);
```

Comment:
Quote around integer values is extra here.

Reply:
fixed in 4d50edad0f236b158ec8fdcb108a3105acbf4749